remsfal-core/pom.xml

```diff
@@ -55,6 +55,16 @@
             <artifactId>microprofile-openapi-api</artifactId>
             <version>${version.open-api}</version>
         </dependency>
+        <dependency>
```

Comment:
Don't use Quarkus dependencies in the core module. Only Jakarta and other specifications are allowed.